### ISSUE TITLE
error  increase, f1 decrease

### ISSUE THREAD

**Issue by Fei-Wang** [kmkurn/pytorch-crf#76]
ner task. 
1. with Bert + softmax+cross-entropy, train loss decrease, f1 increase, and valid data act as train data.
2. with Bert + crf, train loss first decrease and then increase, f1 first increase then decrease. but valid act as normal.

I find it is because I use dropout between Bert and linear layer. when I set model.eval, it is act correct, but if I set model.train, the metric is low. But as I know, dropout is used for avoid overfit, it should't  act so large different between model.eval and model.train.

**Comment by kmkurn** [comment]
Train loss should always decrease if you set the learning rate small enough. Have you tried smaller learning rate? Also, can you check if you can overfit a small portion of the train set?

**Comment by Fei-Wang** [comment]
Thank you for your reply, I will figure it out.
